src/bindings/python/src/pyopenvino/core/compiled_model.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/runtime/compiled_model.hpp"
+
+#include <pybind11/stl.h>
+
+#include "common.hpp"
+#include "pyopenvino/core/containers.hpp"
+#include "pyopenvino/core/infer_request.hpp"
+
+PYBIND11_MAKE_OPAQUE(Containers::TensorIndexMap);
+PYBIND11_MAKE_OPAQUE(Containers::TensorNameMap);
+
+namespace py = pybind11;
+
+void regclass_CompiledModel(py::module m) {
+    py::class_<ov::runtime::CompiledModel, std::shared_ptr<ov::runtime::CompiledModel>> cls(m, "CompiledModel");
```

Comment:
As new class is introduced it need to be accordingly added to init files in openvino python folder and imported as CompiledModel with "base-overloading". As old class is removed all Python related code will be broken due to imports missing.

Please change this to `CompiledModel`:
 https://github.com/openvinotoolkit/openvino/blob/ec6f57872fb4a28aebec37633813f9fb9ca71683/src/bindings/python/src/openvino/runtime/__init__.py#L65
 
 And adjust ie_api.py `CompiledModel as CompiledModelBase` and any other `ExecutableNetwork` to `CompiledModel` accordingly:
 https://github.com/openvinotoolkit/openvino/blob/ec6f57872fb4a28aebec37633813f9fb9ca71683/src/bindings/python/src/openvino/runtime/ie_api.py#L10
 
 **Probably tests and some other components will also need this rename.**

Reply:
Yes I will fix it. At least CI should show all incorrect classes.

Reply:
@jiwaszki fixed